browser/js/common/factories/ReviewFactory.js

```diff
@@ -0,0 +1,17 @@
+app.factory('ReviewFactory', function ($http) {
+  return {
+    createReview: function (stars, review) {
+      return $http.post('/api/v1/reviews', {rating:stars, description:review})
+      .then(function (res) {
+        return res.data;
+      });
+    },
+
+    getReviews: function(songId){
+      return $http.get('/api/v1/reviews')
+      .then(function(res){
+        return res.data;
+      })
+    }
```

Comment:
I figured the admin would need getAllReviews, but def should be getReviewsBySong, but would we want to display all the users Reviews on his account page?